### ISSUE TITLE
Possible bug - can't browse our autogenerated documentation

### ISSUE THREAD

**Issue by buep** [jenkinsci/configuration-as-code-plugin#98]
Related to #46 #96 I can't browse the documentation after #96 is merged.

After a `mvn clean` and `rm -rf work` (just to be sure), I did `mvn hpi:run` and browsed to http://localhost:8080/jenkins/configuration-as-code/ and clicked the link `Documentation`.

The new page opened is empty except from the headline saying "Jenkins Configuration as Code Reference".

**Comment by buep** [comment]
Stacktrace:

```
WARNING: Caught exception evaluating: it.configurators in /jenkins/configuration-as-code/reference. Reason: java.lang.reflect.InvocationTargetException
java.lang.reflect.InvocationTargetException
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.apache.commons.jexl.util.PropertyExecutor.execute(PropertyExecutor.java:125)
	at org.apache.commons.jexl.util.introspection.UberspectImpl$VelGetterImpl.invoke(UberspectImpl.java:314)
	at org.apache.commons.jexl.parser.ASTArrayAccess.evaluateExpr(ASTArrayAccess.java:185)
	at org.apache.commons.jexl.parser.ASTIdentifier.execute(ASTIdentifier.java:75)
	at org.apache.commons.jexl.parser.ASTReference.execute(ASTReference.java:83)
	at org.apache.commons.jexl.parser.ASTReference.value(ASTReference.java:57)
	at org.apache.commons.jexl.parser.ASTReferenceExpression.value(ASTReferenceExpression.java:51)
	at org.apache.commons.jexl.ExpressionImpl.evaluate(ExpressionImpl.java:80)
	at hudson.ExpressionFactory2$JexlExpression.evaluate(ExpressionFactory2.java:74)
	at org.apache.commons.jelly.expression.ExpressionSupport.evaluateRecurse(ExpressionSupport.java:61)
	at org.apache.commons.jelly.expression.ExpressionSupport.evaluateAsIterator(ExpressionSupport.java:94)
	at org.apache.commons.jelly.tags.core.ForEachTag.doTag(ForEachTag.java:89)
	at org.apache.commons.jelly.impl.TagScript.run(TagScript.java:269)
	at org.kohsuke.stapler.jelly.ReallyStaticTagLibrary$1.run(ReallyStaticTagLibrary.java:99)
	at org.apache.commons.jelly.impl.ScriptBlock.run(ScriptBlock.java:95)
	at org.kohsuke.stapler.jelly.ReallyStaticTagLibrary$1.run(ReallyStaticTagLibrary.java:99)
	at org.kohsuke.stapler.jelly.ReallyStaticTagLibrary$1.run(ReallyStaticTagLibrary.java:99)
	at org.kohsuke.stapler.jelly.ReallyStaticTagLibrary$1.run(ReallyStaticTagLibrary.java:99)
	at org.kohsuke.stapler.jelly.ReallyStaticTagLibrary$1.run(ReallyStaticTagLibrary.java:99)
	at org.apache.commons.jelly.impl.ScriptBlock.run(ScriptBlock.java:95)
	at org.kohsuke.stapler.jelly.ReallyStaticTagLibrary$1.run(ReallyStaticTagLibrary.java:99)
	at org.apache.commons.jelly.impl.ScriptBlock.run(ScriptBlock.java:95)
	at org.apache.commons.jelly.tags.core.CoreTagLibrary$2.run(CoreTagLibrary.java:105)
	at org.kohsuke.stapler.jelly.JellyViewScript.run(JellyViewScript.java:95)
	at org.kohsuke.stapler.jelly.DefaultScriptInvoker.invokeScript(DefaultScriptInvoker.java:63)
	at org.kohsuke.stapler.jelly.DefaultScriptInvoker.invokeScript(DefaultScriptInvoker.java:53)
	at org.kohsuke.stapler.jelly.JellyFacet$1.dispatch(JellyFacet.java:97)
	at org.kohsuke.stapler.Stapler.tryInvoke(Stapler.java:715)
	at org.kohsuke.stapler.Stapler.invoke(Stapler.java:845)
	at org.kohsuke.stapler.MetaClass$10.dispatch(MetaClass.java:374)
	at org.kohsuke.stapler.Stapler.tryInvoke(Stapler.java:715)
	at org.kohsuke.stapler.Stapler.invoke(Stapler.java:845)
	at org.kohsuke.stapler.Stapler.invoke(Stapler.java:649)
	at org.kohsuke.stapler.Stapler.service(Stapler.java:238)
	at javax.servlet.http.HttpServlet.service(HttpServlet.java:790)
	at org.eclipse.jetty.servlet.ServletHolder.handle(ServletHolder.java:841)
	at org.eclipse.jetty.servlet.ServletHandler$CachedChain.doFilter(ServletHandler.java:1650)
	at hudson.util.PluginServletFilter$1.doFilter(PluginServletFilter.java:135)
	at hudson.util.PluginServletFilter.doFilter(PluginServletFilter.java:138)
	at org.eclipse.jetty.servlet.ServletHandler$CachedChain.doFilter(ServletHandler.java:1637)
	at hudson.security.csrf.CrumbFilter.doFilter(CrumbFilter.java:49)
	at org.eclipse.jetty.servlet.ServletHandler$CachedChain.doFilter(ServletHandler.java:1637)
	at hudson.security.ChainedServletFilter$1.doFilter(ChainedServletFilter.java:84)
	at hudson.security.ChainedServletFilter.doFilter(ChainedServletFilter.java:90)
	at hudson.security.HudsonFilter.doFilter(HudsonFilter.java:171)
	at org.eclipse.jetty.servlet.ServletHandler$CachedChain.doFilter(ServletHandler.java:1637)
	at org.kohsuke.stapler.compression.CompressionFilter.doFilter(CompressionFilter.java:49)
	at org.eclipse.jetty.servlet.ServletHandler$CachedChain.doFilter(ServletHandler.java:1637)
	at hudson.util.CharacterEncodingFilter.doFilter(CharacterEncodingFilter.java:82)
	at org.eclipse.jetty.servlet.ServletHandler$CachedChain.doFilter(ServletHandler.java:1637)
	at org.kohsuke.stapler.DiagnosticThreadNameFilter.doFilter(DiagnosticThreadNameFilter.java:30)
	at org.eclipse.jetty.servlet.ServletHandler$CachedChain.doFilter(ServletHandler.java:1637)
	at org.eclipse.jetty.servlet.ServletHandler.doHandle(ServletHandler.java:533)
	at org.eclipse.jetty.server.handler.ScopedHandler.handle(ScopedHandler.java:143)
	at org.eclipse.jetty.security.SecurityHandler.handle(SecurityHandler.java:524)
	at org.eclipse.jetty.server.handler.HandlerWrapper.handle(HandlerWrapper.java:132)
	at org.eclipse.jetty.server.handler.ScopedHandler.nextHandle(ScopedHandler.java:190)
	at org.eclipse.jetty.server.session.SessionHandler.doHandle(SessionHandler.java:1595)
	at org.eclipse.jetty.server.handler.ScopedHandler.nextHandle(ScopedHandler.java:188)
	at org.eclipse.jetty.server.handler.ContextHandler.doHandle(ContextHandler.java:1253)
	at org.eclipse.jetty.server.handler.ScopedHandler.nextScope(ScopedHandler.java:168)
	at org.eclipse.jetty.servlet.ServletHandler.doScope(ServletHandler.java:473)
	at org.eclipse.jetty.server.session.SessionHandler.doScope(SessionHandler.java:1564)
	at org.eclipse.jetty.server.handler.ScopedHandler.nextScope(ScopedHandler.java:166)
	at org.eclipse.jetty.server.handler.ContextHandler.doScope(ContextHandler.java:1155)
	at org.eclipse.jetty.server.handler.ScopedHandler.handle(ScopedHandler.java:141)
	at org.eclipse.jetty.server.handler.ContextHandlerCollection.handle(ContextHandlerCollection.java:219)
	at org.eclipse.jetty.server.handler.HandlerCollection.handle(HandlerCollection.java:126)
	at org.eclipse.jetty.server.handler.HandlerWrapper.handle(HandlerWrapper.java:132)
	at org.eclipse.jetty.server.Server.handle(Server.java:564)
	at org.eclipse.jetty.server.HttpChannel.handle(HttpChannel.java:317)
	at org.eclipse.jetty.server.HttpConnection.onFillable(HttpConnection.java:251)
	at org.eclipse.jetty.io.AbstractConnection$ReadCallback.succeeded(AbstractConnection.java:279)
	at org.eclipse.jetty.io.FillInterest.fillable(FillInterest.java:110)
	at org.eclipse.jetty.io.ChannelEndPoint$2.run(ChannelEndPoint.java:124)
	at org.eclipse.jetty.util.thread.Invocable.invokePreferred(Invocable.java:128)
	at org.eclipse.jetty.util.thread.Invocable$InvocableExecutor.invoke(Invocable.java:222)
	at org.eclipse.jetty.util.thread.strategy.EatWhatYouKill.doProduce(EatWhatYouKill.java:294)
	at org.eclipse.jetty.util.thread.strategy.EatWhatYouKill.run(EatWhatYouKill.java:199)
	at org.eclipse.jetty.util.thread.QueuedThreadPool.runJob(QueuedThreadPool.java:672)
	at org.eclipse.jetty.util.thread.QueuedThreadPool$2.run(QueuedThreadPool.java:590)
	at java.lang.Thread.run(Thread.java:748)
Caused by: java.lang.ClassCastException: java.lang.Class cannot be cast to java.lang.reflect.ParameterizedType
	at org.jenkinsci.plugins.casc.Configurator.lookup(Configurator.java:81)
	at org.jenkinsci.plugins.casc.ConfigurationAsCode.listElements(ConfigurationAsCode.java:142)
	at org.jenkinsci.plugins.casc.ConfigurationAsCode.getConfigurators(ConfigurationAsCode.java:128)
	... 86 more
```

**Comment by ewelinawilkosz** [comment]
for me it's:

```
4 errors
16 warnings
[ERROR] Error while creating javadoc report: 
Exit code: 1 - /Users/ewelinawilkosz/praqma/bd_repos/jenkinsci/configuration-as-code-plugin/src/main/java/org/jenkinsci/plugins/casc/Attribute.java:52: warning: no @return
    public boolean isMultiple() {
                   ^
/Users/ewelinawilkosz/praqma/bd_repos/jenkinsci/configuration-as-code-plugin/src/main/java/org/jenkinsci/plugins/casc/Attribute.java:76: warning: no @return
    public List<String> possibleValues() {
                        ^
/Users/ewelinawilkosz/praqma/bd_repos/jenkinsci/configuration-as-code-plugin/src/main/java/org/jenkinsci/plugins/casc/Configurator.java:182: warning: no @return
    public abstract Class<T> getTarget();
                             ^
/Users/ewelinawilkosz/praqma/bd_repos/jenkinsci/configuration-as-code-plugin/src/main/java/org/jenkinsci/plugins/casc/Configurator.java:187: warning: no @return
    public Class getExtensionPoint() {
                 ^
/Users/ewelinawilkosz/praqma/bd_repos/jenkinsci/configuration-as-code-plugin/src/main/java/org/jenkinsci/plugins/casc/Configurator.java:197: warning: no @return
    public String getExtensionSource() throws IOException {
                  ^
/Users/ewelinawilkosz/praqma/bd_repos/jenkinsci/configuration-as-code-plugin/src/main/java/org/jenkinsci/plugins/casc/Configurator.java:197: warning: no @throws for java.io.IOException
    public String getExtensionSource() throws IOException {
                  ^
/Users/ewelinawilkosz/praqma/bd_repos/jenkinsci/configuration-as-code-plugin/src/main/java/org/jenkinsci/plugins/casc/Configurator.java:209: warning: no @return
    public String getDisplayName() { return ""; }
                  ^
/Users/ewelinawilkosz/praqma/bd_repos/jenkinsci/configuration-as-code-plugin/src/main/java/org/jenkinsci/plugins/casc/Configurator.java:224: warning: no @throws for java.lang.Exception
    public abstract T configure(Object config) throws Exception;
                      ^
/Users/ewelinawilkosz/praqma/bd_repos/jenkinsci/configuration-as-code-plugin/src/main/java/org/jenkinsci/plugins/casc/Configurator.java:229: warning: no @return
    public List<Attribute> getAttributes() {
                           ^
/Users/ewelinawilkosz/praqma/bd_repos/jenkinsci/configuration-as-code-plugin/src/main/java/org/jenkinsci/plugins/casc/Configurator.java:249: warning: no @return
    public abstract Set<Attribute> describe();
                                   ^
/Users/ewelinawilkosz/praqma/bd_repos/jenkinsci/configuration-as-code-plugin/src/main/java/org/jenkinsci/plugins/casc/Configurator.java:253: error: malformed HTML
     * FIXME would prefer <st:include page="help-${a.name}.html" class="${c.target}" optional="true"/>
                          ^
/Users/ewelinawilkosz/praqma/bd_repos/jenkinsci/configuration-as-code-plugin/src/main/java/org/jenkinsci/plugins/casc/Configurator.java:253: error: bad use of '>'
     * FIXME would prefer <st:include page="help-${a.name}.html" class="${c.target}" optional="true"/>
                                                                                                     ^
/Users/ewelinawilkosz/praqma/bd_repos/jenkinsci/configuration-as-code-plugin/src/main/java/org/jenkinsci/plugins/casc/Configurator.java:255: warning: no @param for attribute
    public String getHtmlHelp(String attribute) throws IOException {
                  ^
/Users/ewelinawilkosz/praqma/bd_repos/jenkinsci/configuration-as-code-plugin/src/main/java/org/jenkinsci/plugins/casc/Configurator.java:255: warning: no @return
    public String getHtmlHelp(String attribute) throws IOException {
                  ^
/Users/ewelinawilkosz/praqma/bd_repos/jenkinsci/configuration-as-code-plugin/src/main/java/org/jenkinsci/plugins/casc/Configurator.java:255: warning: no @throws for java.io.IOException
    public String getHtmlHelp(String attribute) throws IOException {
                  ^
/Users/ewelinawilkosz/praqma/bd_repos/jenkinsci/configuration-as-code-plugin/src/main/java/org/jenkinsci/plugins/casc/ConfigurationAsCode.java:112: warning: no @param for in
    public static void configure(InputStream in) throws Exception {
                       ^
/Users/ewelinawilkosz/praqma/bd_repos/jenkinsci/configuration-as-code-plugin/src/main/java/org/jenkinsci/plugins/casc/ConfigurationAsCode.java:112: warning: no @throws for java.lang.Exception
    public static void configure(InputStream in) throws Exception {
                       ^
/Users/ewelinawilkosz/praqma/bd_repos/jenkinsci/configuration-as-code-plugin/src/main/java/org/jenkinsci/plugins/casc/SecretSource.java:22: error: invalid use of @param
     * @param secret
       ^
/Users/ewelinawilkosz/praqma/bd_repos/jenkinsci/configuration-as-code-plugin/src/main/java/org/jenkinsci/plugins/casc/SecretSource.java:22: warning: no description for @param
     * @param secret
       ^
/Users/ewelinawilkosz/praqma/bd_repos/jenkinsci/configuration-as-code-plugin/src/main/java/org/jenkinsci/plugins/casc/SecretSource.java:23: error: invalid use of @return
     * @return the revealed secret. Null in the case that the implementation is not replacing anything. Throws exception
       ^

Command line was: /Library/Java/JavaVirtualMachines/jdk1.8.0_111.jdk/Contents/Home/jre/../bin/javadoc @options @packages

Refer to the generated Javadoc files in '/Users/ewelinawilkosz/praqma/bd_repos/jenkinsci/configuration-as-code-plugin/target/site/apidocs' dir.

org.apache.maven.reporting.MavenReportException: 
Exit code: 1 - /Users/ewelinawilkosz/praqma/bd_repos/jenkinsci/configuration-as-code-plugin/src/main/java/org/jenkinsci/plugins/casc/Attribute.java:52: warning: no @return
    public boolean isMultiple() {
                   ^
/Users/ewelinawilkosz/praqma/bd_repos/jenkinsci/configuration-as-code-plugin/src/main/java/org/jenkinsci/plugins/casc/Attribute.java:76: warning: no @return
    public List<String> possibleValues() {
                        ^
/Users/ewelinawilkosz/praqma/bd_repos/jenkinsci/configuration-as-code-plugin/src/main/java/org/jenkinsci/plugins/casc/Configurator.java:182: warning: no @return
    public abstract Class<T> getTarget();
                             ^
/Users/ewelinawilkosz/praqma/bd_repos/jenkinsci/configuration-as-code-plugin/src/main/java/org/jenkinsci/plugins/casc/Configurator.java:187: warning: no @return
    public Class getExtensionPoint() {
                 ^
/Users/ewelinawilkosz/praqma/bd_repos/jenkinsci/configuration-as-code-plugin/src/main/java/org/jenkinsci/plugins/casc/Configurator.java:197: warning: no @return
    public String getExtensionSource() throws IOException {
                  ^
/Users/ewelinawilkosz/praqma/bd_repos/jenkinsci/configuration-as-code-plugin/src/main/java/org/jenkinsci/plugins/casc/Configurator.java:197: warning: no @throws for java.io.IOException
    public String getExtensionSource() throws IOException {
                  ^
/Users/ewelinawilkosz/praqma/bd_repos/jenkinsci/configuration-as-code-plugin/src/main/java/org/jenkinsci/plugins/casc/Configurator.java:209: warning: no @return
    public String getDisplayName() { return ""; }
                  ^
/Users/ewelinawilkosz/praqma/bd_repos/jenkinsci/configuration-as-code-plugin/src/main/java/org/jenkinsci/plugins/casc/Configurator.java:224: warning: no @throws for java.lang.Exception
    public abstract T configure(Object config) throws Exception;
                      ^
/Users/ewelinawilkosz/praqma/bd_repos/jenkinsci/configuration-as-code-plugin/src/main/java/org/jenkinsci/plugins/casc/Configurator.java:229: warning: no @return
    public List<Attribute> getAttributes() {
                           ^
/Users/ewelinawilkosz/praqma/bd_repos/jenkinsci/configuration-as-code-plugin/src/main/java/org/jenkinsci/plugins/casc/Configurator.java:249: warning: no @return
    public abstract Set<Attribute> describe();
                                   ^
/Users/ewelinawilkosz/praqma/bd_repos/jenkinsci/configuration-as-code-plugin/src/main/java/org/jenkinsci/plugins/casc/Configurator.java:253: error: malformed HTML
     * FIXME would prefer <st:include page="help-${a.name}.html" class="${c.target}" optional="true"/>
                          ^
/Users/ewelinawilkosz/praqma/bd_repos/jenkinsci/configuration-as-code-plugin/src/main/java/org/jenkinsci/plugins/casc/Configurator.java:253: error: bad use of '>'
     * FIXME would prefer <st:include page="help-${a.name}.html" class="${c.target}" optional="true"/>
                                                                                                     ^
/Users/ewelinawilkosz/praqma/bd_repos/jenkinsci/configuration-as-code-plugin/src/main/java/org/jenkinsci/plugins/casc/Configurator.java:255: warning: no @param for attribute
    public String getHtmlHelp(String attribute) throws IOException {
                  ^
/Users/ewelinawilkosz/praqma/bd_repos/jenkinsci/configuration-as-code-plugin/src/main/java/org/jenkinsci/plugins/casc/Configurator.java:255: warning: no @return
    public String getHtmlHelp(String attribute) throws IOException {
                  ^
/Users/ewelinawilkosz/praqma/bd_repos/jenkinsci/configuration-as-code-plugin/src/main/java/org/jenkinsci/plugins/casc/Configurator.java:255: warning: no @throws for java.io.IOException
    public String getHtmlHelp(String attribute) throws IOException {
                  ^
/Users/ewelinawilkosz/praqma/bd_repos/jenkinsci/configuration-as-code-plugin/src/main/java/org/jenkinsci/plugins/casc/ConfigurationAsCode.java:112: warning: no @param for in
    public static void configure(InputStream in) throws Exception {
                       ^
/Users/ewelinawilkosz/praqma/bd_repos/jenkinsci/configuration-as-code-plugin/src/main/java/org/jenkinsci/plugins/casc/ConfigurationAsCode.java:112: warning: no @throws for java.lang.Exception
    public static void configure(InputStream in) throws Exception {
                       ^
/Users/ewelinawilkosz/praqma/bd_repos/jenkinsci/configuration-as-code-plugin/src/main/java/org/jenkinsci/plugins/casc/SecretSource.java:22: error: invalid use of @param
     * @param secret
       ^
/Users/ewelinawilkosz/praqma/bd_repos/jenkinsci/configuration-as-code-plugin/src/main/java/org/jenkinsci/plugins/casc/SecretSource.java:22: warning: no description for @param
     * @param secret
       ^
/Users/ewelinawilkosz/praqma/bd_repos/jenkinsci/configuration-as-code-plugin/src/main/java/org/jenkinsci/plugins/casc/SecretSource.java:23: error: invalid use of @return
     * @return the revealed secret. Null in the case that the implementation is not replacing anything. Throws exception
       ^

Command line was: /Library/Java/JavaVirtualMachines/jdk1.8.0_111.jdk/Contents/Home/jre/../bin/javadoc @options @packages

Refer to the generated Javadoc files in '/Users/ewelinawilkosz/praqma/bd_repos/jenkinsci/configuration-as-code-plugin/target/site/apidocs' dir.

	at org.apache.maven.plugin.javadoc.AbstractJavadocMojo.executeJavadocCommandLine(AbstractJavadocMojo.java:5188)
	at org.apache.maven.plugin.javadoc.AbstractJavadocMojo.executeReport(AbstractJavadocMojo.java:2075)
	at org.apache.maven.plugin.javadoc.JavadocReport.generate(JavadocReport.java:130)
	at org.apache.maven.plugin.javadoc.JavadocReport.execute(JavadocReport.java:318)
	at org.apache.maven.plugin.DefaultBuildPluginManager.executeMojo(DefaultBuildPluginManager.java:134)
	at org.apache.maven.lifecycle.internal.MojoExecutor.execute(MojoExecutor.java:208)
	at org.apache.maven.lifecycle.internal.MojoExecutor.execute(MojoExecutor.java:154)
	at org.apache.maven.lifecycle.internal.MojoExecutor.execute(MojoExecutor.java:146)
	at org.apache.maven.lifecycle.internal.MojoExecutor.executeForkedExecutions(MojoExecutor.java:353)
	at org.apache.maven.lifecycle.internal.MojoExecutor.execute(MojoExecutor.java:198)
	at org.apache.maven.lifecycle.internal.MojoExecutor.execute(MojoExecutor.java:154)
	at org.apache.maven.lifecycle.internal.MojoExecutor.execute(MojoExecutor.java:146)
	at org.apache.maven.lifecycle.internal.LifecycleModuleBuilder.buildProject(LifecycleModuleBuilder.java:117)
	at org.apache.maven.lifecycle.internal.LifecycleModuleBuilder.buildProject(LifecycleModuleBuilder.java:81)
	at org.apache.maven.lifecycle.internal.builder.singlethreaded.SingleThreadedBuilder.build(SingleThreadedBuilder.java:51)
	at org.apache.maven.lifecycle.internal.LifecycleStarter.execute(LifecycleStarter.java:128)
	at org.apache.maven.DefaultMaven.doExecute(DefaultMaven.java:309)
	at org.apache.maven.DefaultMaven.doExecute(DefaultMaven.java:194)
	at org.apache.maven.DefaultMaven.execute(DefaultMaven.java:107)
	at org.apache.maven.cli.MavenCli.execute(MavenCli.java:993)
	at org.apache.maven.cli.MavenCli.doMain(MavenCli.java:345)
	at org.apache.maven.cli.MavenCli.main(MavenCli.java:191)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.codehaus.plexus.classworlds.launcher.Launcher.launchEnhanced(Launcher.java:289)
	at org.codehaus.plexus.classworlds.launcher.Launcher.launch(Launcher.java:229)
	at org.codehaus.plexus.classworlds.launcher.Launcher.mainWithExitCode(Launcher.java:415)
	at org.codehaus.plexus.classworlds.launcher.Launcher.main(Launcher.java:356)
[INFO] 
```

and http://localhost:8080/jenkins/plugin/configuration-as-code/ gives me

```
HTTP ERROR 404

Problem accessing /jenkins/plugin/configuration-as-code/. Reason:

    Not Found
```

can it be both for the same reason?

**Comment by ewelinawilkosz** [comment]
I was accessing the wrong link, so errors and warnings above are still an issue but I can access page with documentation, it's just like Bue says, page is empty except from the headline saying "Jenkins Configuration as Code Reference".

**Comment by MadsNielsen** [comment]
``` 
[INFO] Session node0nmoxt7idd8491qj558gnkykve1 already being invalidated
Feb 12, 2018 2:09:36 PM org.jenkinsci.plugins.casc.Configurator lookup
WARNING: Configuration-as-Code can't handle type class java.util.HashMap
Feb 12, 2018 2:09:36 PM org.jenkinsci.plugins.casc.Configurator lookup
WARNING: Configuration-as-Code can't handle type class java.util.HashMap
Feb 12, 2018 2:09:36 PM org.jenkinsci.plugins.casc.Configurator lookup
WARNING: Configuration-as-Code can't handle type class jenkins.branch.MultiBranchProjectViewHolder$ViewImpl
Feb 12, 2018 2:09:36 PM org.jenkinsci.plugins.casc.Configurator lookup
WARNING: Configuration-as-Code can't handle type class jenkins.branch.OrganizationFolderViewHolder$ViewImpl
Feb 12, 2018 2:09:36 PM org.jenkinsci.plugins.casc.Configurator lookup
WARNING: Configuration-as-Code can't handle type class hudson.security.LegacySecurityRealm
Feb 12, 2018 2:09:36 PM org.jenkinsci.plugins.casc.Configurator lookup
WARNING: Configuration-as-Code can't handle type class hudson.security.ProjectMatrixAuthorizationStrategy
Feb 12, 2018 2:09:36 PM org.jenkinsci.plugins.casc.Configurator lookup
WARNING: Configuration-as-Code can't handle type class jenkins.branch.MultiBranchProjectViewHolder$ViewImpl
Feb 12, 2018 2:09:36 PM org.jenkinsci.plugins.casc.Configurator lookup
WARNING: Configuration-as-Code can't handle type class jenkins.branch.OrganizationFolderViewHolder$ViewImpl
Feb 12, 2018 2:09:36 PM org.jenkinsci.plugins.casc.Configurator lookup
WARNING: Configuration-as-Code can't handle type class java.util.HashMap
Feb 12, 2018 2:09:36 PM org.jenkinsci.plugins.casc.Configurator lookup
WARNING: Configuration-as-Code can't handle type class java.util.HashSet
Feb 12, 2018 2:09:36 PM org.jenkinsci.plugins.casc.Configurator lookup
WARNING: Configuration-as-Code can't handle type class java.util.HashSet
Feb 12, 2018 2:09:36 PM org.jenkinsci.plugins.casc.Configurator lookup
WARNING: Configuration-as-Code can't handle type class java.util.HashSet
Feb 12, 2018 2:09:36 PM org.jenkinsci.plugins.casc.Configurator lookup
WARNING: Configuration-as-Code can't handle type class java.util.HashSet
```

**Comment by lanwen** [comment]
fixed in #95 
https://github.com/jenkinsci/configuration-as-code-plugin/pull/95/files#diff-410e45fabc6e62bed9a91d95d0f40037R80

```diff
-        if (Collection.class.isAssignableFrom(clazz)) {
+        if (Collection.class.isAssignableFrom(clazz) && type instanceof ParameterizedType) {
```

will rebase and send as separate PR if #111 will be merged (contains test with new styled annotation)

**Comment by ewelinawilkosz** [comment]
merged! any chance we can get this PR @lanwen? :)

**Comment by MadsNielsen** [comment]
Already fixed on master: 

https://github.com/jenkinsci/configuration-as-code-plugin/blob/4050973c11d66ba734d2286a19c46173739858c9/src/main/java/org/jenkinsci/plugins/casc/Configurator.java#L80-L93

It was resolved by https://github.com/jenkinsci/configuration-as-code-plugin/pull/105
